pyocd_pemicro/pemicro_probe.py

```diff
@@ -58,7 +59,7 @@ class PEMicroProbe(DebugProbe):
     APSEL_SHIFT = 24
     APSEL_APBANKSEL = APSEL | APBANKSEL
 
-    ## Part of the error message for exceptions raised when there isn't a PEMicro library
```

Comment:
A double hash at the start of a comment marks the comment as a documentation comment that will be picked up by Doxygen.